build_tools/python/e2e_test_framework/models/utils.py

```diff
@@ -0,0 +1,44 @@
+## Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Utilities to help define models."""
+
+import string
+from typing import Dict, Sequence
+
+from e2e_test_framework.definitions import common_definitions
+
+
+def generate_batch_models(
+    id_template: str, name_template: str, tags: Sequence[str],
+    source_type: common_definitions.ModelSourceType, source_url_template: str,
+    entry_function: str, input_type_templates: Sequence[str],
+    batch_sizes: Sequence[int]) -> Dict[int, common_definitions.Model]:
+  """Generate model definitions for different batch sizes by substituting
```

Comment:
Nit: IMO multiple arguments per line like this is really hard to read. I think if you add that trailing comma it will auto-format to one per line

Reply:
Done.